libs/gl-client/src/signer/resolve.rs

```diff
@@ -32,7 +32,7 @@ impl Resolver {
             Message::GetChannelBasepoints(_) => true,
             Message::ValidateCommitmentTx(_) => true,
             Message::SignWithdrawal(_) => true,
-            Message::ReadyChannel(_) => true,
+            Message::SetupChannel(_) => true,
```

Comment:
We probably want to update issue #276 which refers to the outdated `ReadyChannel` when this merges

Reply:
Excellent catch :+1: